packages/design-system/package.json

```diff
@@ -14,8 +14,8 @@
   "types": "dist/types/index.d.ts",
   "files": [
     "dist",
-    "src/components/**/*.jsx",
-    "!src/components/**/*.test.jsx"
+    "src/components/**/*.?sx",
```

Comment:
This is just a file pattern, so we can be specific here like this:
 `"src/components/**/*.{jsx,tsx}",`

Reply:
Good point.